### DISCUSSION TITLE
i3wm over VNC connection

### DISCUSSION THREAD

**Discussion by Joshua-Micheletti** [i3/i3#6114] [Q&A]
I'm trying to run tightvncserver in Ubuntu 20.04 launching i3 in its xstartup script. Everything works, i see the i3-status bar. They keybinds though are completely wrong, when i press ALT + 1 it goes to desktop 9, ALT + 2 makes it go to desktop 10 etc...

I couldn't figure out any other wrong controls cause i can't even open a terminal with ALT + Enter.

The configuration is the default i3 configuration with ALT as the mod key.

I tried with different environments (XFCE4, bspwm, dwm) and the keybinds there all work fine.

I saw some questions on the internet about this behaviour but they stayed unanswered for years.

Does anyone have a clue as to why it behaves like this?

**Comment by ScaryAardvark** [comment]
I have exactly the same problem. Did you manage to find a solution?

**Comment by ScaryAardvark** [comment]
I think I may have found a solution. Do not use tightvnc. I changed to using tigervnc and it worked like a charm
